common/src/main/java/tachyon/worker/BlockStoreLocation.java

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
The concern is correct. Originally, I put this class in common because I thought client may also need this class in case we change client side code and make client face with "real" worker interface. Currently, the worker is still exposing the current old worker interface in thrift RPC to make worker module compatible with other existing component.

I am open to move this class to server module, if it turns out only needed by master and worker.